index.js

```diff
@@ -936,20 +936,34 @@ class GithubScm extends Scm {
      * @param  {String}   config.type      Can be 'pr' or 'repo'
      * @param  {Object}   config.payload   The webhook payload received from the SCM service.
      * @param  {String}   config.token     Service token to authenticate with Github
+     * @param  {String}   config.scmUri    The scmUri to get PR info of
```

Comment:
```suggestion
     * @param  {String}   [config.scmUri]    The scmUri to get PR info of
```
to indicate that the param is optional